src/contrib/cluster/Akka.DistributedData.Tests/WriteAggregatorSpec.cs

```diff
@@ -85,7 +85,7 @@ public WriteAckAdapter(IActorRef replica)
         private readonly Address _nodeB = new Address("akka.tcp", "Sys", "b", 2552);
         private readonly Address _nodeC = new Address("akka.tcp", "Sys", "c", 2552);
         private readonly Address _nodeD = new Address("akka.tcp", "Sys", "d", 2552);
-        private readonly IImmutableSet<Address> _nodes;
+        private readonly IImmutableList<Address> _nodes;
```

Comment:
Why the change from set to list?

Reply:
because the base class `ReadWriteAggregator` changes of `nodes` from `IImmutableSet` to `IImmutableList` to keep the order of nodes by oldest for the `PreferOldest` scenario.

Reply:
Got it, thank you